client/package.json

```diff
@@ -33,5 +36,10 @@
     "webpack": "^1.12.14"
   },
   "devDependencies": {
+    "eslint": "^3.10.2",
+    "eslint-config-shakacode": "^13.1.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^2.2.3",
+    "eslint-plugin-react": "^6.7.1"
```

Comment:
@robwise They are not in the generator. Are they @nostophilia? 

@nostophilia was instructed to use the linter to be sure what we generated would run clean.